src/main/java/com/iota/iri/network/pipeline/BroadcastPayload.java

```diff
@@ -0,0 +1,44 @@
+package com.iota.iri.network.pipeline;
+
+import com.iota.iri.controllers.TransactionViewModel;
+import com.iota.iri.network.neighbor.Neighbor;
+
+import java.util.Optional;
+
+/**
+ * Defines a payload which gets submitted to the {@link BroadcastStage}.
+ */
+public class BroadcastPayload {
```

Comment:
I would have turned a blind eye if you would have used a shortcut like the `Pair` object

Custom class is more readable in the end so :+1:

Reply:
:ok_hand: